apps/web/src/pages/api/v1/data/[projectId]/index.ts

```diff
@@ -89,6 +89,19 @@ export default async function getWeightsHandler(
 
     const duration = performance.now() - now;
 
+    if (
+      typeof req.query.projectId === "string" &&
+      req.query.projectId.endsWith(".js")
+    ) {
+      const jsContent = `window.${ABBY_WINDOW_KEY} = ${JSON.stringify(
```

Comment:
Maybe we should sanitize this at some point, but this could be done in a seperate PR, if we want to quickly merge this now. You should just trust your colleagues :D